### ISSUE TITLE
Rule proposal: selector-solitary-root

### ISSUE THREAD

**Issue by davidtheclark** [stylelint/stylelint#130]
`:root` shouldn't be grouped or combined with other selectors.

Bad:
```css
:root,
body { /* ... */ }

body:root { /* ... */ }
```

Good:
```css
:root { /* ... */ }
body { /* ... */ }
```

Thoughts?

I couldn't think of a better name right now ... `selector-no-grouped-or-combined-root` would be the most accurate, I think, but rambly.

**Comment by davidtheclark** [comment]
Or maybe it could be `root-no-other-selectors`?

**Comment by tunnckoCore** [comment]
:+1: for that in title, last is the worst for me lol :)

**Comment by MoOx** [comment]
+1 for the rule

For the name maybe something like selector-root-no-composition ?
We can use some vocabulary from selector structure http://dev.w3.org/csswg/selectors-4/#structure

**Comment by davidtheclark** [comment]
I like that (`selector-root-no-composition`). Good idea. I'll add it to #1.

**Comment by jeddy3** [comment]
> `selector-root-no-composition`

Great name!
